src/content/docs/distributed-tracing/ui-data/related-trace-entity-signals.mdx

```diff
@@ -90,11 +63,41 @@ You may override the default compare with behavior by changing the value of the
 
 If there is no tracing data preserved in the comparison window, the distributed trace insights subheader will not contain a "compared with" clause.
 
-## When no tracing data is present
-If your entity is not reporting trace data, the entity insights component will present a "no data" view which includes links to our docs and a Guided Install setup.
+## Additional actions
+
+Click the gear icon at the top-right of the component for additional actions:
+* **View all** to see a full-page view of this list when many signals are being displayed.
+* **Guided install** walks you through the steps to enable distributed tracing for this service.
+* **Instrumentation status** lists the distributed tracing configuration status for all of your services.
+* **See our docs** is a link to this page.
+
 <img
-  title="No trace data pattern"
-  alt="View when there is no trace data present"
-  src={noTraceDataPattern}
+  title="Signal summaries example"
+  alt="Example of signal summaries view"
+  src={signalInsightsWithControls}
+/>
+
+## Full-page View
+This view shows the same list of significant signals, with more real estate to scroll through longer lists.
+It also includes a filter option to focus on signals of a particular type.
+
+<img
+  title="Signal summaries example"
+  alt="Example of signal summaries view"
+  src={signalInsightsFullPage}
 />
 
+## No significant changes presented
+If no upstream or downstream services had a significant changes in their performance impact, the all clear view is shown.
+
+<img
+  title="Signal summaries example"
+  alt="Example of signal summaries view"
+  src={signalSummariesWithAllClear}
+/>
+
+## When no tracing data is present
+You will see one of the following messages:
+* **No data for this time range** When viewing an older time window for your service that's beyond the retention period of your trace data. Select a time within the last week to see distributed tracing insights.
+* **No data from your entity** If your service is not reporting distributed tracing data. Follow links to our docs or the Guided Install to setup distributed tracing.
```

Comment:
```suggestion
## Understanding Missing Tracing Data Messages [#no-trace-data]

To ensure you're working with the most relevant distributed tracing insights, here's a breakdown of the messages you might encounter when data is unavailable, along with recommended actions:

* `No data for this time range`: 
     * This message indicates that you've selected a time window for your service that falls outside the current retention period for trace data.
     * To access insights, adjust the time range to focus on a period within the last week.
* `No data from your entity`:
     * This message suggests that your service isn't currently reporting distributed tracing data.
     * To unlock these valuable insights, set up distributed tracing for your service by following the [guided install](https://onenr.io/0OQM6lV3DwG). If you're new to distributed tracing, start by reading the [Distributed tracing: Planning guide](https://docs.newrelic.com/docs/distributed-tracing/concepts/distributed-tracing-planning-guide/).
```

Reply:
@jvanpeltnr  How do they enable distributed tracing on the dependencies of their service? Is it the same process as enabling DT for the service?

Reply:
Yes, same general process. We do have a page where you can see the config status for all apps you have access to, but you still have to know which entities are the dependencies for yours to use it, so the docs are just as useful in that case.

Reply:
@jvanpeltnr How does my suggestion look ^

Reply:
looks good - thanks!